src/Business Foundation/App/NoSeriesCopilot/Permissions/NoSeriesCopilotObjects.permissionset.al

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+permissionset 330 "No. Series Copilot - Objects"
+{
+    Access = Internal;
+    Assignable = false;
+    Permissions = 
+        codeunit "No. Series Cop. Add Intent"=X,
+        codeunit "No. Series Cop. Change Intent"=X,
+        codeunit "No. Series Cop. Tools Impl."=X,
+        codeunit "No. Series Copilot Impl."=X,
+        codeunit "No. Series Copilot Install"=X,
+        codeunit "No. Series Copilot Register"=X,
+        codeunit "No. Series Copilot Upgr. Tags"=X,
+        page "No. Series Copilot Setup"=X,
+        tabledata "No. Series Copilot Setup"=RIMD;
```

Comment:
the *objects permission sets should not contain tabledata. These permissionsets exist as building blocks for those customers that do not use <type> * = X in their permissionsets which means they need a way to run the internal objects, and this is it. 
The tabledata permissions should be split by usage scenario, but the defaults we use are Read, View, Edit or Admin 
 where not all are required and we typically have Read + 1 or two more. Each subsequent level includes the previous. As some of these might seem ambiguous, here's how I see them:
 
 Read = only r or R.
 View = ability to directly view the data on screen but no direct edit ability (optional indirect imd).
 Edit = direct edit on pages

Reply:
Sorry, this one is my bad. I had the permission set generated to please CodeCop, and then I accidentally checked in the permission set before actually getting this right. I will revisit the permissions!